torch/_dynamo/variables/builder.py

```diff
@@ -964,13 +971,13 @@ def wrap_tensor(self, value: torch.Tensor):
     def wrap_numpy_ndarray(self, value):
         assert isinstance(value, np.ndarray)
 
-        source = self.get_source()
+        source = NumpyTensorSource(self.get_source())
         tensor_value = torch.as_tensor(value)
```

Comment:
Let's move everything to `as_tensor`. Note that that's what's being done here, and it's what we want to do really